ksz8863/src/ksz8863_ctrl.c

```diff
@@ -20,8 +20,7 @@
 #define KSZ8863_SPI_LOCK_TIMEOUT_MS 500
 
 typedef struct {
-    i2c_port_t i2c_port;
-    uint8_t dev_addr;
+    i2c_master_dev_handle_t dev_handle;
 } ksz8863_i2c_spec_t;
```

Comment:
Seems `ksz8863_i2c_spec_t` and `ksz8863_spi_spec_t` are no more used. Should be removed then.

Reply:
Removed.